v3/plugins/redhat/rhamt/0.0.8/meta.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v2
+id: redhat.rhamt-vscode-extension
+version: 0.0.8
+type: VS Code extension
+name: rhamt
+displayName: Red Hat Application Migration Toolkit
+title: Red Hat Application Migration Toolkit
+description: Application Migration and Modernization using RHAMT
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+publisher: redhat
+repository: https://github.com/windup/rhamt-vscode-extension
+category: Other
+firstPublicationDate: "2019-06-19"
+spec:
+  endpoints:
+   - name: "configuration-endpoint"
+     public: true
+     targetPort: 61436
+     attributes:
+       protocol: http
+  - name: "report-endpoint"
+    public: true
+    targetPort: 61435
+    attributes:
+       protocol: http
+  containers:
+    - image: "johnsteele/rhamt-vscode-extension:java8"
```

Comment:
We usually avoid using images from personal repos. Can''t you push the image on a project repository or use one of the existing images from another plugin?

Reply:
I've moved the image to the windup3 org.